src/Connection.php

```diff
@@ -96,18 +114,31 @@ public function receiveSubscriptionMessage(string $topicName, ReceiveMessageOpti
     private function createSubscription(string $topicName): void
     {
         if ($this->checkSubscriptionExists($topicName)) {
+            if ($this->cache) {
+                $this->cache->set('subscription_'.$topicName.'_'.$this->subscriptionName, true, 3600);
```

Comment:
Find a better solution to check the topic / subscription key. are tel configurable.